src/main.py

```diff
@@ -17,14 +19,15 @@ def initialize():
 if __name__ == "__main__":
     
     ### Main code goes here###
+    test_mode = bool(input("Activate Test Mode (True/False):"))
```

Comment:
Did some looking to this. bool casting makes empty strings false and non-empty string true no matter what they are. I think theres a library that converts "true" -> True (bool). Try to change it to that